### DISCUSSION TITLE
Welcome to PS2toAmiga Discussions!

### DISCUSSION THREAD

**Discussion by Jartza** [Jartza/PS2toAmiga#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by pocoaust** [comment]
Hi mate,
I have assembled 2 of these and for the life of me, cant get them to work.  Could you pls send me a raw BIN file.  When I write the file and then do a verify, it fails on avrdude.

I hae source attiny85 from 2 sources, same thing.

Thanks
Paul

**Reply by Jartza** [reply]
> Hi mate, I have assembled 2 of these and for the life of me, cant get them to work. Could you pls send me a raw BIN file. When I write the file and then do a verify, it fails on avrdude.
> 
> I hae source attiny85 from 2 sources, same thing.
> 
> Thanks Paul

Hi!

I have only ever tried this with Arduino IDE and [ATTinyCore](https://github.com/SpenceKonde/ATTinyCore) myself. The important part is also selection of the settings mentioned in README:


    Board: ATtiny24/45/85 (No bootloader)
    Chip: ATtiny85
    Clock Source: 16 MHz (PLL)
    TImer 1 Clock: CPU frequency
    LTO: Enabled
    millis(): Enabled

...After which you have to select "burn bootloader". Of course on this small ATtiny, it doesn't actually burn bootloader, it sets the fuses to correct chip speeds. After that you need to upload the compiled HEX to your chip.

I can check if I still have the .hex on my computer, but you still must set the chip fuses before trying it as those Attinys ship from factory with 1MHz internal clock enabled. Some programmers fail on AVRDUDE due to that reason and you need to slow down the clock for the fuse burning with -B parameter, for example `-B 8` will slow down programmer clock usually enough.

After the fuses have been programmed to 16MHz internal clock, faster speeds for actual binary programming are not an issue.

//Jartza
